### PR TITLE
Various case and text fixes.

### PR DESCRIPTION
**Pull request by alimony**
This fixes some consistency issues with text cases on labels (https://github.com/bluesky-social/social-app/issues/709) and includes a few other tiny text polish fixes. I will make line comments in this PR to clarify a few things.

### CODE REVIEW

File: src/view/com/modals/AddAppPasswords.tsx
Comment (alimony) [inline src/view/com/modals/AddAppPasswords.tsx:1]: This seems like a jumble of two different versions, I used the one used elsewhere in the app.

File: src/view/com/modals/DeleteAccount.tsx
Comment (alimony) [inline src/view/com/modals/DeleteAccount.tsx:76]: Wording seems clearer to me.

File: src/view/screens/SearchMobile.tsx
Comment (alimony) [inline src/view/screens/SearchMobile.tsx:167]: To clarify this searches for both users and posts.

File: src/view/screens/Settings.tsx
Comment (alimony) [inline src/view/screens/Settings.tsx:357]: To follow the convention of system menu items etc. where ellipsis means there will be further prompts before any action is actually taken.

File: src/view/com/auth/SplashScreen.web.tsx
Comment (ansh) [inline src/view/com/auth/SplashScreen.web.tsx:50]: Can we keep this lowercase?

File: src/view/com/auth/SplashScreen.tsx
Comment (ansh) [inline src/view/com/auth/SplashScreen.tsx:36]: Can we keep this lowercase?

File: __e2e__/tests/home-screen.test.ts
Comment (ansh) [inline __e2e__/tests/home-screen.test.ts:75]: Let's revert this for now

File: src/view/com/modals/AddAppPasswords.tsx
Comment (ansh) [inline src/view/com/modals/AddAppPasswords.tsx:187]: Good catch! Thanks :)

File: src/view/shell/desktop/RightNav.tsx
Comment (ansh) [inline src/view/shell/desktop/RightNav.tsx:49]: Let's revert this for now.

**Review by ansh — CHANGES_REQUESTED** [review]
Mostly looks good. Have a few changes.

**Review by ansh — CHANGES_REQUESTED** [review]
Mostly looks good. Have a few changes.

**Comment by ansh** [comment]
@alimony Fixed lint errors on main. Feel free to merge main into this branch to re-runt he tests.

**Comment by alimony** [comment]
@ansh Done and passing!

**Comment by alimony** [comment]
@ansh This is still waiting for approval after making your requested changes.

**Review by ansh — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**
